modules/appeals_api/spec/requests/v2/supplemental_claims_controller_spec.rb

```diff
@@ -70,9 +70,9 @@ def base_path(path)
     end
 
     context 'noticeAcknowledgement' do
-      it 'benefitType = compensation and noticeAcknowledgement = false' do
+      it "benefitType = compensation and noticeAcknowledgement = 'no'" do
```

Comment:
This ripple effect of changing the type really isn't feeling like the right solution, and goes against the fact that there's already a type (boolean) that takes care of this for us.

If this change was necessary to make rswag happy, then it should be done at the rswag conversion level - we shouldn't be breaking schema conventions just to appease our testing tool.

Reply:
The answer is yes, but it wasn't a boolean before, either.

I'll go back and try again with a boolean, since it really feels like it should work. Also, just a note, not that this changed my opinion about the boolean value, but the form is two distinct checkboxes - one for yes, and one for no

Reply:
Resolved!